src/Libraries/DSOffice/DSOffice.csproj

```diff
@@ -10,8 +10,8 @@
     <AssemblyName>DSOffice</AssemblyName>
     <DocumentationFile>$(OutputPath)\$(UICulture)\DSOffice.xml</DocumentationFile>
   </PropertyGroup>
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
-        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1000">
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
```

Comment:
any idea what the difference is between these packages?

Reply:
1000 (2016) can use Office 2013 Excel interop
1001 (2022) can use for Excel 2013/2016/2019 COM interop
1001 also supports net standard 2